src/translations/nbTranslations.json

```diff
@@ -1274,6 +1276,8 @@
         "funding_id": "Tilskudds-ID",
         "funding_name": "Tilskuddsnavn",
         "funding_sum": "Sum",
+        "funding_without_project": "Finansiering uten prosjekt",
+        "funding_without_project_description": "<0>Mottar forskningen finansiering som ikke er tilknyttet et prosjekt så trykk på \"Legg til ny finansiering\" og deretter søk etter finansiør.</0> <0>Ikke registrer finsiering som er tilknyttet et prosjekt. Finansiering tilknyttet prosjekt legges til når du legger til prosjekt lengre opp på siden. Hvis ikke informasjonen om finansiering vises så må dette registreres på prosjektet i NVA.</0>",
```

Comment:
```suggestion
        "funding_without_project_description": "<0>Mottar forskningen finansiering som ikke er tilknyttet et prosjekt så trykk på \"Legg til ny finansiering\" og deretter søk etter finansiør.</0> <0>Ikke registrer finansiering som er tilknyttet et prosjekt. Finansiering tilknyttet prosjekt legges til når du legger til prosjekt lengre opp på siden. Hvis ikke informasjonen om finansiering vises så må dette registreres på prosjektet i NVA.</0>",
```

Reply:
Teksten samsvarer ikke med knappen. Dette ble fikset for alle(?) knapper før helga så kanskje en stund siden du har pulla?
![bilde](https://github.com/user-attachments/assets/cb2299e1-db7a-41e1-9c27-f580828527bd)